test/executor.test.js

```diff
@@ -450,6 +450,38 @@ describe('executor', function() {
       });
     });
 
+    it('should parse simple config variable from env var', function(done) {
+      process.env.restApiRoot = '/url-from-env-var';
+      boot.execute(app, simpleMiddlewareConfig('routes',
+        { path: '${restApiRoot}' }
+      ));
+
+      supertest(app).get('/url-from-env-var').end(function(err, res) {
+        if (err) return done(err);
+        expect(res.body.path).to.equal('/url-from-env-var');
+        delete process.env.restApiRoot;
+        done();
+      });
+    });
+
+    it('dynamic variable from `env var` should have' +
+    ' precedence over app.get()', function(done) {
+      process.env.restApiRoot = '/url-from-env-var';
```

Comment:
@bajtos I was removing `process.env` after the test upon success, was thinking the next set of test might get polluted as I am doing it on the sub-describe. Would I need to also need to handle it after? or It's okay for next set of test to handle its own environment.